### DISCUSSION TITLE
Welcome to home-assistant-HomeWhiz Discussions!

### DISCUSSION THREAD

**Discussion by TechHummel** [home-assistant-HomeWhiz/home-assistant-HomeWhiz#201] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Croft5** [comment]
Hi, just got a new Grundig dishwasher and found that all the elements are unavailable. I see this is a known issue. I look forward to the solution. Thank you for your work :-)

**Comment by parvu** [comment]
Hi, just found that Grundig 7292146991 side by side fridge is working with this integration. 17 entities found.
